src/components/UI/SamplesMap/index.tsx

```diff
@@ -43,13 +44,70 @@ const SamplesMap: React.FC<MapProps> = ({ samples }) => {
   const sampleInfoWindow = useRef(new google.maps.InfoWindow());
   const clusterInfoWindow = useRef(new google.maps.InfoWindow());
   const newBoundary = useRef(new google.maps.LatLngBounds());
-
   const markers = useRef({});
+  const [markingEezRegions, setMarkingEezRegions] = useState(false);
+
+  // eslint-disable-next-line consistent-return
+  const fetchPolygonCoordinates = async () => {
+    try {
+      const response = await axios.get(
+        `https://marineregions.org/rest/getGazetteerGeometries.ttl/${samples[0].attributes.mrgid}/`
```

Comment:
Is the Turtle format used over JSON-LD for any particular reason? Just wondering if the substring logic below would be any easier with the JSON-LD version?

Reply:
@SandyRogers I wasn't aware there was a JSON-LD format when I implemented this. 
It was quite tricky to get the polygons drawing correctly. I'm afraid changing it to work with the JSON-LD format now may have a lot of ramifications that might take more time to fix.